plugin/markdown/plugin.js

```diff
@@ -415,6 +428,73 @@ const Plugin = () => {
 
 	}
 
+	function parseFrontMatter (content, options) {
+		options = getSlidifyOptions( options)
+
+		const parsedFrontMatter = fm(content)
+
+		content = parsedFrontMatter.body;
+		if (parsedFrontMatter.frontmatter){
+			options.metadata = yaml.load(parsedFrontMatter.frontmatter);
+			if (!('slideType' in options.metadata)) {
+				content = `Missing "slideType" in default metadata`
+				console.error(content)
+				delete options.metadata
```

Comment:
should delete the metadata or provide the default required data to it?

Reply:
Deleting the metadata from options, in case of missing slide type. Otherwise, the presence of invalid metadata in options causes a problem.